packages/core/src/components/forms/numericInput.tsx

```diff
@@ -151,7 +151,12 @@ export interface INumericInputProps extends IIntentProps, IProps {
     onButtonClick?(valueAsNumber: number, valueAsString: string): void;
 
     /** The callback invoked when the value changes due to typing, arrow keys, or button clicks. */
-    onValueChange?(valueAsNumber: number, valueAsString: string, inputElement: HTMLInputElement | null): void;
+    onValueChange?(
+        valueAsNumber: number,
+        valueAsString: string,
+        inputElement: HTMLInputElement | null,
+        instance: NumericInput,
```

Comment:
please revert this change unrelated to #4200

Reply:
Tried your fork and apparently it works correctly, so I'm removing instance from this PR and will try again on official version once this and the other PR is merged